pvacseq/lib/pipeline.py

```diff
@@ -57,6 +57,10 @@ def tsv_file_path(self):
     def convert_vcf(self):
         print("Converting VCF to TSV")
         sys.stdout.flush()
+        if os.path.exists(self.tsv_file_path()):
+            print("TSV file already exists. Skipping.")
+            return
```

Comment:
`def status_message` seems easy enough to implement in a future PR. I don't think a logging framework is necessary since the status messages only get outputted by this module.